src/connectors/relevant-sort/connectRelevantSort.ts

```diff
@@ -77,11 +93,14 @@ const connectRelevantSort: RelevantSortConnector = function connectRelevantSort(
 
         const { appliedRelevancyStrictness } = results || {};
 
+        const isVirtualReplica = appliedRelevancyStrictness !== undefined;
+
         return {
           isRelevantSorted:
             typeof appliedRelevancyStrictness !== 'undefined' &&
             appliedRelevancyStrictness > 0,
-          isVirtualReplica: appliedRelevancyStrictness !== undefined,
+          isVirtualReplica,
+          canRefine: isVirtualReplica && results?.nbHits! > 0,
```

Comment:
Even if `nbSortedHits === 0`, there can be a possibility to see different results with different `relevancyStrictness`. So we're checking it with `nbHits`, not with `nbSortedHits`.

Reply:
I wonder if just `isVirtualReplica` isn't enough here, can't you in theory still swap between virtual and regular view of something with 0 hits? On the other hand, having no hits is probably a good condition to hide it

Reply:
Umm right. We can swap between virtual and regular, so even if it's not virtual, as long as nbHits > 0, it should be able to refine. So I'm thinking of just `results?.nbHits! > 0`. Makes sense to you?

Reply:
actually thought about it again, and I think the current behaviour is correct. Can someone confirm? @francoischalifour @shortcuts

Reply:
> so even if it's not virtual, as long as nbHits > 0, it should be able to refine.

But refine rely on the `relevancyStrictness`, so we should check if it's a virtual replica anyway, no?

I think the current implementation make sense

Reply:
Oh right. `isVirtualReplica` is true when it's 0, too. So it seems alright to me.